http-server/src/main/java/io/airlift/http/server/HttpServer.java

```diff
@@ -384,6 +388,9 @@ private static ServletContextHandler createServletContext(Servlet theServlet,
             context.setErrorHandler(handler);
         }
 
+        // allow encoded slashes to occur in URI paths
+        context.getServletHandler().setDecodeAmbiguousURIs(true);
```

Comment:
Can this cause any security issues?

Reply:
The only thing it skips are `UriCompliance.AMBIGUOUS_VIOLATIONS`. So, it seems safe.